src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1274,6 +1320,22 @@ def _public_key_bytes(self, encoding, format, evp_pkey, cdata):
         assert res == 1
         return self._read_mem_bio(bio)
 
+    def _parse_asn1_time(self, asn1_time):
```

Comment:
Ok, I can wait for this to go back first. Then I'll rebase my changes for final review.